common/environment/environment.hpp

```diff
@@ -66,7 +66,13 @@ enum EnvMachineOS
     MachineOsSize = 4
 };
 
-
+enum RoxieTargetType
+{
+    RTTUnknown = 0,
+    RTTQueries = 1,
```

Comment:
RTTQueries isn't very descriptive - I struggled to work out what it meant.  I think RTTDeployed may be more descriptive, or something else?  @afishbeck ?

Reply:
@ghalliday @wangkx Not sure of the best name, but the thing that makes roxie queries special is not whether they are published/deployed or not.
A query can be published to hthor or thor... if you do so you will see it when you browse to eclqueries:8002 for example.  And you can run it with a simple SOAP or REST call to eclqueries:8002 as well.

The thing that makes a roxie query special is that it runs like a service that can be called directly, in real time.

If you call a published query on hthor, thor, or even "workunit-roxie" queries get cloned as new workunit "jobs" and placed on the queue.

I guess my point is that if we want to be truly correct then the names would be something like:

RTTCallable and RTTSubmit (RTTSubmittable)
or
RTTRealtime and RTTQueued

But of course we could be less precise and use the terms based on most peoples perception which would be

RTTDeployed/RTTPublished and RTTQueued.

Reply:
I like RTTPublished and RTTQueued. @ghalliday ?